### ISSUE TITLE
Creating getcomment API

### ISSUE THREAD

**Issue by ibrahimbayat** [bounswe/2021SpringGroup2#81]
I will write getcomment api

**Comment by ibrahimbayat** [comment]
As I stated in issue #53 it is was very hard for me to use sqlalchemy and docker tools in my api because I am very new to these programs. In these api with given post id I read a comment from the database. To check this api I needed to add a comment to comments table and it takes almost two hours for me. I think we used many new thing in this project, also using these in the same project make thing more worse for me. After adding a comment to the comment table. I can get it with my api. I also used postman to test this api.

**Comment by ibrahimbayat** [comment]
My friends though that we should take the parameter as eventid not post id. Then I tried to change it according to this change in design decision. Then I requested pull for the branch of this function. But my function was returning just the first comment of event. So, I needed to change it according to reviewers comments. Also I needed to change returning value because object from the database can not be in json format. I needed to change to this format manually.  Then we merged branch "getcommentibrahim" with  "development" branch.
